lib/Test2/Formatter/TeamCity.pm

```diff
@@ -408,7 +619,7 @@ sub _event_other {
     my $self  = shift;
     my $event = shift;
 
-    die "Unexpected event! $event";
+    # die "Unexpected event! $event";
```

Comment:
See ca3b1dbc855589e6859e0c2c79db31e75aa6fa08 for the explanation.  Basically this is Dave's old code that I commented out.  It should be removed.